mp4parse/src/lib.rs

```diff
@@ -439,6 +445,7 @@ pub struct TrackEncryptionBox {
 #[derive(Debug, Default, Clone)]
 pub struct ProtectionSchemeInfoBox {
     pub code_name: String,
+    pub scheme_type: Option<SchemeTypeBox>,
```

Comment:
Wasn't sure if there's a convention around naming such members. I considered naming this similar to `tenc`, but prefer the more verbose naming for those who are not familiar with the FourCC. Am happy to change to `schm` if it's more fitting.